extensions/data-transfer/portability-data-transfer-google/src/main/java/org/datatransferproject/datatransfer/google/media/GoogleMediaExporter.java

```diff
@@ -222,8 +226,18 @@ private ExportResult<MediaContainerResource> exportPhotosContainer(
         continue;
       }
 
-      photosBuilder.add(GoogleMediaItem.convertToPhotoModel(Optional.empty(), googleMediaItem));
+      try {
+        photosBuilder.add(GoogleMediaItem.convertToPhotoModel(Optional.empty(), googleMediaItem));
+      } catch(ParseException e) {
+        monitor.info(() -> "Parse exception occurred while converting photo, skipping this item. "
+            + "Failure message : %s ", e.getMessage());
+
+        errors.add(GoogleErrorLogger.createErrorDetail(
+            googleMediaItem.getId(), googleMediaItem.getFilename(), e, /* canSkip= */ true));
+      }
```

Comment:
wait, so this method 

-> turns a list of PhotoModels into GoogleMediaItems
-> turns the new GoogleMediaItems into PhotoModels

why are we even doing this transformation?

Reply:
Not sure. Will not modify this here but will send the fix in the next and add dtp folks to make sure there isn't something that we might be missing.